worker/echo/main.go

```diff
@@ -16,6 +16,16 @@ import (
 var yggdDispatchSocketAddr string
 
 func main() {
+	// Set log level inherited from the environment
+	if logLevel, ok := os.LookupEnv("YGG_LOG_LEVEL"); ok {
+		level, err := log.ParseLevel(logLevel)
+		if err != nil {
+			log.Fatalf("cannot parse log level: %v", err)
+		}
+		log.SetLevel(level)
+	}
+	log.Debug(os.Environ())
```

Comment:
Shouldn't be this debug print introduced with some log message? Something like: "current environment variables"?

Reply:
Yes, this is a good approach.